app/electron.js

```diff
@@ -21,8 +21,8 @@ const staticWindows = {
   },
   help : {
     config : {
-      height          : 625,
-      width           : 625,
+      height          : 400,
+      width           : 550,
```

Comment:
yeah, no problem I was thinking the same, but was not sure if we want to add more info now.